### ISSUE TITLE
ModuleNotFoundError: No module named 'pwd'

### ISSUE THREAD

**Issue by qjwzmy** [XuehaiPan/nvitop#12]
#### Runtime Environment

- Operating system and version: Windows 10 Build 19042.685
- Python version: `3.8.10`
- Terminal emulator and version: `PyCharm 2021.3 (Professional Edition)`
- `nvitop` version or commit: `0.5.2`
- `python-ml-py` version: `11.450.51`
- Locale: `zh_CN.UTF-8`

#### Current Behavior

When I type the `nvitop` in console in pycharm, it repports the error `ModuleNotFoundError: No module named 'pwd'`. 
I have run `nvitop` correctly when I install it.  

#### Expected Behavior

I want to use it for its great gui.

#### Steps to Reproduce

1.In PyCharm IDE, switch to `console` and type `nvitop`, and enter.

#### Traceback

```pytb
Traceback (most recent call last):
  File "C:\Users\user\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 194, in _run_module_as_main
  File "C:\Users\user\AppData\Local\Programs\Python\Python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\PyPrj\venv\Scripts\nvitop.exe\__main__.py", line 4, in <module>
  File "d:\pyprj\venv\lib\site-packages\nvitop\cli.py", line 12, in <module>
    from nvitop.gui import Top, Device, libcurses, colored, USERNAME
  File "d:\pyprj\venv\lib\site-packages\nvitop\gui\__init__.py", line 6, in <module>
    from nvitop.gui.library import Device, libcurses, colored, USERNAME, SUPERUSER
  File "d:\pyprj\venv\lib\site-packages\nvitop\gui\library\__init__.py", line 15, in <module>
    from nvitop.gui.library.utils import (colored, cut_string, make_bar,
  File "d:\pyprj\venv\lib\site-packages\nvitop\gui\library\utils.py", line 66, in <module>
    USERNAME = getpass.getuser()
  File "C:\Users\user\AppData\Local\Programs\Python\Python38\lib\getpass.py", line 168, in getuser
    import pwd
ModuleNotFoundError: No module named 'pwd'
```

**Comment by XuehaiPan** [comment]
@qjwzmy Thanks for the report!

> When I type the `nvitop` in console in pycharm, it repports the error `ModuleNotFoundError: No module named 'pwd'`.

I cannot reproduce this on my machine (PyCharm Terminal Powershell, PyCharm Terminal CMD, and PyCharm Python Console). Do you use a shell program other than above (e.g. `Git-Bash`)?

Could you try the following command in your terminal?

```bash
python -c 'import getpass; print(getpass.getuser())'
```

and

```bash
python -c 'import os; print(os.getlogin())'
```

------

In `getpass` (Python's built-in module):

```python
def getuser():
    """Get the username from the environment or password database.

    First try various environment variables, then the password
    database.  This works on Windows as long as USERNAME is set.

    """

    for name in ('LOGNAME', 'USER', 'LNAME', 'USERNAME'):
        user = os.environ.get(name)
        if user:
            return user

    # If this fails, the exception will "explain" why
    import pwd
    return pwd.getpwuid(os.getuid())[0]
```

When your terminal emulator doesn't set `USER` nor `USERNAME`, Python will try to import `pwd` (Unix-like system only) to get the current user name.

**Comment by XuehaiPan** [comment]
@qjwzmy Hi, I add a new commit to address this ([dev@56db1b](56db1bf0004aedcd88f726d05945ccf101034d23)).

Could you try to reinstall `nvitop` with:

```bash
pip3 install git+https://github.com/XuehaiPan/nvitop.git@dev#egg=nvitop
```

**Comment by XuehaiPan** [comment]
The potential fix (56db1bf0004aedcd88f726d05945ccf101034d23) is now available in version 0.5.2.1 on PyPI.

Thanks for the feedback from @qjwzmy! Feel free to reopen this issue if you have any other questions.
